libs/libarchfpga/src/arch_util.cpp

```diff
@@ -495,13 +495,12 @@ void alloc_and_load_default_child_for_pb_type(t_pb_type* pb_type,
     copy->num_pb = 1;
 
     /* Power */
-    copy->pb_type_power = (t_pb_type_power*)vtr::calloc(1,
-                                                        sizeof(t_pb_type_power));
+    copy->pb_type_power = new t_pb_type_power();
```

Comment:
calloc 0-initializes the data it allocates.  Looking at the code in physical_types.h, is no default constructor (i.e. the constructor with no arguments) for t_pb_type_power defined. Hence the compiler will make one. That constructor does not initialize built in types like integers and floats. 

Hence this code change is dangerous: we have gone from a block of 0-initialized memory to a struct that has unknown state.  Probably @jgoeders used calloc on purpose to initialize the power elements to 0.

To convert calloc uses like this to new, you have to make a default constructor for the struct/class in question and initialize all its data members to 0.